app/src/main/java/nerd/tuxmobil/fahrplan/congress/extensions/TextViewExtensions.kt

```diff
@@ -17,3 +21,19 @@ var TextView.textOrHide: CharSequence
             isVisible = true
         }
     }
+
+/**
+ * Set the given [plainLinkUrl] and the optional [urlTitle] as clickable link to this [TextView].
+ */
+@JvmOverloads
+fun TextView.setLinkText(
+        plainLinkUrl: String,
+        urlTitle: String? = null,
+        movementMethod: MovementMethod,
+        @ColorInt linkTextColor: Int,
+) {
+    val formattedUrl = plainLinkUrl.toLinkified(urlTitle).toSpanned()
```

Comment:
There's no need to create HTML only to convert it to a `Spanned` right away.

```suggestion
    val linkText = urlTitle ?: plainLinkUrl
    val formattedUrl = SpannableString(linkText)
    formattedUrl.setSpan(URLSpan(plainLinkUrl), 0, linkText.length, Spanned.SPAN_EXCLUSIVE_EXCLUSIVE)
```

Reply:
Thank you for the hint. :heavy_check_mark: I simplified it in ca3c00b44466295c77588d8a2e7b3adaeecfa485.

Reply:
@cketti Is this good for you? Did you spot any other issues?

Reply:
I only gave it a quick look. LGTM